src/components/pages/contact/Contact.jsx

```diff
@@ -24,25 +21,31 @@ export const Contact = () => {
             }, (error) => {
                 console.log(error.text);
             });
+    }
 
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        setFormValues(FORM_INITIAL_VALUES);
+        sendEmail();
```

Comment:
chyba lepiej wyczyscic formularz po wysllaniu o ile nie wystapi error, jakby mi wyskoczyl alert ze nie wyslano maila i musialabym pisac na nowo to bym byla zla

Reply:
nie wysyłasz wgl pustego maila jeżeli czyścisz formularz linike przed? 🤔